demonstrations/tutorial_kernel_module.py

```diff
@@ -0,0 +1,596 @@
+r"""Quantum Embedding Kernels with the kernels module
+=====================================================
+
+.. meta::
+    :property="og:description": Kernels and alignment training with Pennylane.
+    :property="og:image": https://pennylane.ai/qml/_images/QEK_thumbnail.png
+
+.. related::
+
+    tutorial_kernel_based_training Kernel-based training with scikit-learn
+    tutorial_data_reuploading_classifier Classification with data reuploading
+
+*Authors: Peter-Jan Derks, Paul Fährmann, Elies Gil-Fuster, Tom
+Hubregtsen, Johannes Jakob Meyer and David Wierichs. Posted: 24 June 2021*
+
+Kernel methods are one of the cornerstones of classical machine
+learning. To understand what a kernel method does, let’s first revisit
+one of the simplest methods to assign binary labels to datapoints:
+linear classification.
+
+Imagine we want to discern two different classes of points that lie in
+different corners of the plane. A linear classifier corresponds to
+drawing a line and assigning different labels to the regions on opposing
+sides of the line:
+
+.. figure:: ../demonstrations/kernel_module/linear_classification.png
+   :align: center
+   :width: 30%
+
+We can mathematically formalize this by assigning the label :math:`y`
+via
+
+.. math::
+   y(\boldsymbol{x}) = \operatorname{sgn}(\langle \boldsymbol{w}, \boldsymbol{x}\rangle + b).
+
+The vector :math:`\boldsymbol{w}` points perpendicular to the line and
+thus determine its slope. The independent term :math:`b` specificies the
+position on the plane. In this form, linear classification can also be
+extended to higher dimensional vectors :math:`\boldsymbol{x}`, where a
+line does not divide the entire space into two regions anymore. Instead
+one needs a *hyperplane*. It is immediately clear that this method is
+not very powerful, as datasets that are not separable by a hyperplane
+can’t be treated.
```

Comment:
```suggestion
can’t be classified without error.
```
Maybe more accurate?

Reply:
(throughout this file): I recommend replacing unicode quotation marks with ASCII ones:
```suggestion
can't be treated.
```

There have been issues in the past with users on Windows attempting to run the demos, but experiencing invalid encoding errors.